package.json

```diff
@@ -13,9 +20,20 @@
   "files": [
     "frontend"
   ],
+  "scripts": {
+    "prepare": "npm run build",
```

Comment:
not a big deal, but I'd like to stick with the convention of using Yarn rather than NPM

Reply:
Actually these just run the build script…any reason we need `prepare` and `prepack`?

Reply:
These are purely for me running from my personal repo. They can be removed. They're just a lifecycle hook for git repos! Sorry!